README.md

```diff
@@ -86,56 +86,47 @@ molecule test
 # Developing
 
 ## Create a new role
-In all the steps below substitue your role name for `example`
+In all the steps below substitue your role name for `your_new_role`
 1. Initialize the role with molecule.
    Run the following command from the root of this repo:
 
    ```bash
-   molecule init role -r roles/example
+   molecule init role roles/your_new_role --driver-name docker
    ```
 1. Set up to run from github actions `vi .github/workflows/molecule_tests.yml` add for your role at the end matrix of the roles
    ```
-       - example
+       - your_new_role
    ```
 1. Setup the directory to run molecule
 
    1. copy over the root molecule.yml
       ```bash
-      cp roles/example/molecule.yml roles/example/molecule/default
-      cp roles/example/yaml-lint.yml roles/example/molecule/default
-      cp roles/example/main.yml roles/example/meta
-      cp roles/example/.ansible-lint roles/example/.ansible-lint
+      cp roles/example/molecule.yml roles/your_new_role/molecule/default
+      cp roles/example/.ansible-lint roles/your_new_role/.ansible-lint
       ```
 
-   1. edit `roles/example/meta/main.yml` and change `to include your role name`
+   1. edit `roles/example/meta/main.yml` and change `your_new_role`
```

Comment:
Not sure what this means. maybe just delete and use the "add a description" line below